kivymd/uix/picker.py

```diff
@@ -448,6 +456,7 @@ class MDDatePicker(
     _sel_day_widget = ObjectProperty()
     cal_list = None
     cal_layout = ObjectProperty()
+    cal_radius = NumericProperty(0)
```

Comment:
You do not need to use the prefix `cal` for the `radius` parameter.

Why did you add the "radius" parameter only for the `MDDatePicker` class. I think it will be more logical if we make a common class `BaseDialog` (https://github.com/kivymd/KivyMD/blob/master/kivymd/uix/dialog.py#L180) for the `MDTimePicker`, `MDDatePicker`, `MDThemePicker` classes and specify the radius parameter in the `BaseDialoe` class.

Reply:
`radius = ListProperty([7, 7, 7, 7])` if you mean this I can't use that because I only change 2 corners everytime example: `radius: [x, x, 0, 0]` or `radius: [x, 0 0, x]` etc because calendar is 2 different canvas's. The radius in BaseDialog returns all 4 corners radius. Correct me if I didn't understand correctly. I agree it would be better to have one radius for time, date and theme pickers. I didn't think of that.

Reply:
Yes, we will not be able to inherit from one `BaseDialog`. In this case, we need to add the `radius` parameter to all three classes `MDTimePicker`, `MDDatePicker`, `MDThemePicker`.

Reply:
actually I found a way, check the last commit.

Reply:
I meant that we have to add the radius parameter to the rest of the classes - `MDTimePicker`, `MDThemePicker`.

Reply:
Done. Let me know if that's better.